one-acre-fund/ussd-sms
# Healthy path

Involved tickets
## [SER-10](https://oneacrefund.atlassian.net/browse/SER-10)
**KE and RW**
(THIS SERVICE IS triggered externally) to test it we need to pay hence trigger it. so it means we will just use button trigger service to be able to see its response to the trigger operation
As a client, I want to see my distance from the healthy path in repayment receipts, so that I know if I'm on track or not

__KE__
https://telerivet.com/p/0c6396c9/service/SVb2e575732d72adb3/edit
- Add your phone number to the simulator phone number so that you can receive the sms (include +country code)
- click simulate service
- you should receive an sms with the repayment information including the healthy path data.

__RW__
- https://telerivet.com/p/8799a79f/service/SV41751ceed72268e9/edit
- click simulate service
- you should receive an sms with the repayment information including the healthy path data.

## [SER-13](https://oneacrefund.atlassian.net/browse/SER-13)
**KE**
As a client, I want to be able to see my distance from the healthy path when I check my balance, so that I know how I stand with repayment - KE Config

_testing_
- dial in the short code
- add the account number **(26215997)**
- choose balance on the menu **(option 2)**
- on the balance information, there should be the the healthy path line too.

## [SER-14](https://oneacrefund.atlassian.net/browse/SER-14)
**RW**
As a client, I want to be able to see my distance from the healthy path when I check my balance, so that I know how I stand with repayment - RW Config

_testing_
- dial in the short code
- add account number **(24046873)**
- choose balance on the menu **(option 1)**
-  on the balance information, there should be the the healthy path line too. (in kinyarwanda though)
